crates/net/eth-wire/src/types/broadcast.rs

```diff
@@ -204,10 +207,84 @@ pub struct NewPooledTransactionHashes68 {
     pub hashes: Vec<H256>,
 }
 
+impl Encodable for NewPooledTransactionHashes68 {
+    fn length(&self) -> usize {
+        #[derive(RlpEncodable)]
+        struct EncodableNewPooledTransactionHashes68 {
+            types: Bytes,
+            sizes: Vec<usize>,
+            hashes: Vec<H256>,
+        }
```

Comment:
Bytes is just a wrapped Vec<u8>,

I think we have two options here

1. replace `types: Vec<u8>` with `types: Bytes` and we can keep all derives
2. keep custom impl but use references instead of cloning

I guess the only argument for Vec<u8> would be that all fields are Vec but can also go with Bytes

Reply:
If we use `Bytes`, we lose the ability to derive `Arbitrary` because `Bytes` does not implement it. So I'm leaning towards keeping the custom impl and using references instead of cloning